### PR TITLE
KAA-366: Build instruction for x86-64 platform (C++ SDK).

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

File: client/client-multi/client-cpp/README.txt [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+************************************
+INSTALL
+************************************
+
+To build the C++ endpoint SDK, do the following:
+
+1. Download and untar the Kaa C++ SDK archive.
+2. Run the following commands:
+    avrogen.sh
```
Comment (izeigerman) [inline client/client-multi/client-cpp/README.txt]: Please use "./avrogen.sh" instead.

File: client/client-multi/client-cpp/README.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+PLATFORM DEPEDENCIES
+************************************
+There are no dependencies on platforms.
+
+
+************************************
+BUILD EXAMPLE
+************************************
+The following example illustrates the build procedure with the INFO log level and the LOGGING module:
+    avrogen.sh
```
Comment (izeigerman) [inline client/client-multi/client-cpp/README.txt]: Please use "./avrogen.sh" instead.

**Comment by izeigerman** [comment]
LGTM

File: client/client-multi/client-cpp/README.txt [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    mkdir build
+    cd build
+    cmake -DKAA_INSTALL_PATH='/home/username/kaa' -DKAA_DEBUG_ENABLED=1 -DKAA_MAX_LOG_LEVEL=4 -DKAA_WITHOUT_EVENTS=1 ..
+    make
+    make install
+
+
```
Comment (ghost) [inline client/client-multi/client-cpp/README.txt]: Please, remove redundant new lines.

File: client/client-multi/client-cpp/README.txt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+************************************
+BUILD EXAMPLE
+************************************
+The following example illustrates the build procedure with the INFO log level and the LOGGING module:
+    avrogen.sh
+    mkdir build
+    cd build
+    cmake -DKAA_INSTALL_PATH='/home/username/kaa' -DKAA_DEBUG_ENABLED=1 -DKAA_MAX_LOG_LEVEL=4 -DKAA_WITHOUT_EVENTS=1 ..
```
Comment (ghost) [inline client/client-multi/client-cpp/README.txt:113]: The Cmake's parameter list doesn't match to the description of the example.

File: client/client-multi/client-cpp/CMakeLists.txt [hunk 5]
```diff
@@ -15,21 +15,76 @@
...
+# Sets common compilation flags.
+# 
+# NOTE: Platform specific compilation flags should be set 
+# in the corresponding platform CMake script(s).
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+
+if(KAA_DEBUG_ENABLED)
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -ggdb")
+else()
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -os -s")
```
Comment (ghost) [inline client/client-multi/client-cpp/CMakeLists.txt]: Please, replace the "-os" flag with "-Os" to be complied with compiler documentation.

**Pull request merged**